<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWorkPageSet</class>
 <widget class="QWidget" name="QWorkPageSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1912</width>
    <height>874</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QWorkPageSet</string>
  </property>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>812</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="buttons" stdset="0">
    <string notr="true">flat</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>1010</width>
     <height>542</height>
    </rect>
   </property>
   <property name="title">
    <string>工作</string>
   </property>
   <property name="borders" stdset="0">
    <string notr="true">group_tit</string>
   </property>
   <widget class="QCheckBox" name="checkBox_transferOpen">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>380</y>
      <width>950</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>手工传输弹窗（提示：关闭后，保留上一次手工传输设置内容）</string>
    </property>
    <property name="checks" stdset="0">
     <string notr="true">push</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>332</y>
      <width>621</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>弹窗设置</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>721</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>功能按键</string>
    </property>
   </widget>
   <widget class="QWidget" name="" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>252</y>
      <width>660</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>660</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>660</width>
      <height>16777215</height>
     </size>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="checkBox_delete">
       <property name="minimumSize">
        <size>
         <width>210</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>210</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_recomput">
       <property name="minimumSize">
        <size>
         <width>210</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>210</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>重新计算</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_transferOpen_2">
       <property name="minimumSize">
        <size>
         <width>210</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>210</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>视觉识别</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>177</y>
      <width>880</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>880</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>880</width>
      <height>16777215</height>
     </size>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="checkBox_transfer">
       <property name="minimumSize">
        <size>
         <width>210</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>210</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>手工传输</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_export">
       <property name="minimumSize">
        <size>
         <width>210</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>210</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>导出</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_patient">
       <property name="minimumSize">
        <size>
         <width>210</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>210</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>患者信息</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_recheck">
       <property name="minimumSize">
        <size>
         <width>210</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>210</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>复查</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
    <zorder>checkBox_export</zorder>
    <zorder>checkBox_transfer</zorder>
    <zorder>checkBox_recheck</zorder>
   </widget>
   <widget class="QWidget" name="" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>103</y>
      <width>880</width>
      <height>40</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="checkBox_select">
       <property name="minimumSize">
        <size>
         <width>210</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>210</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>筛选</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_detail">
       <property name="minimumSize">
        <size>
         <width>210</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>210</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>结果详情</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_audit">
       <property name="minimumSize">
        <size>
         <width>210</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>210</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>审核</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_print">
       <property name="minimumSize">
        <size>
         <width>210</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>210</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>打印</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>509</y>
     <width>1010</width>
     <height>279</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="borders" stdset="0">
    <string notr="true">group_tit</string>
   </property>
   <widget class="QPushButton" name="pushButton_up">
    <property name="geometry">
     <rect>
      <x>833</x>
      <y>134</y>
      <width>116</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>向上</string>
    </property>
    <property name="autoRepeatDelay">
     <number>1000</number>
    </property>
    <property name="autoRepeatInterval">
     <number>1000</number>
    </property>
    <property name="buttons" stdset="0">
     <string notr="true">btn_dlg</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>103</y>
      <width>773</width>
      <height>176</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>245</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>44</number>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_down">
    <property name="geometry">
     <rect>
      <x>833</x>
      <y>198</y>
      <width>116</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>向下</string>
    </property>
    <property name="buttons" stdset="0">
     <string notr="true">btn_dlg</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>500</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>样本列表</string>
    </property>
   </widget>
   <widget class="QFrame" name="line_h">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>103</y>
      <width>1010</width>
      <height>1</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lines" stdset="0">
     <string notr="true">solid</string>
    </property>
   </widget>
   <widget class="QFrame" name="line_v">
    <property name="geometry">
     <rect>
      <x>772</x>
      <y>103</y>
      <width>1</width>
      <height>176</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lines" stdset="0">
     <string notr="true">solid</string>
    </property>
   </widget>
   <widget class="QFrame" name="line_v_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>103</y>
      <width>1</width>
      <height>176</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lines" stdset="0">
     <string notr="true">solid</string>
    </property>
   </widget>
   <widget class="QFrame" name="line_h_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>278</y>
      <width>1010</width>
      <height>1</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lines" stdset="0">
     <string notr="true">solid</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>1019</x>
     <y>5</y>
     <width>888</width>
     <height>783</height>
    </rect>
   </property>
   <property name="title">
    <string>监控页面</string>
   </property>
   <property name="borders" stdset="0">
    <string notr="true">group_tit</string>
   </property>
   <widget class="QCheckBox" name="checkBox_call">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>60</y>
      <width>800</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>监控页面定时呼出</string>
    </property>
    <property name="checks" stdset="0">
     <string notr="true">push</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>751</width>
      <height>44</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>无操作</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>15</number>
       </property>
       <item>
        <widget class="QLineEdit" name="lineEdit_time">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>40</height>
          </size>
         </property>
         <property name="maxLength">
          <number>8</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>分钟后显示</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
